### PR TITLE
add results page

### PR DESCRIPTION
**Pull request by monstermunchkin**

### CODE REVIEW

File: src/libclient/result/resultmodel.cpp [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    q->beginInsertRows(QModelIndex(), position, position);
+    identToResult.insert(TaskId(result.value("task_id").toInt()), position);
+    results = resultScheduler->results();
+    q->endInsertRows();
+}
+
+void ResultModel::Private::resultModified(const QVariantMap &result)
+{
+    int pos = identToResult.value(TaskId(result.value("task_id").toInt()));
+    Q_ASSERT(pos != -1);
```
Comment (MKV21) [inline src/libclient/result/resultmodel.cpp]: Please use the logging facility and don't quit the program with an assertion (fix this also in the ReportModel).

File: src/libclient/result/resultmodel.cpp [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        return 0;
+    }
+
+    return d->results.size();
+}
+
+int ResultModel::columnCount(const QModelIndex &parent) const
+{
+    Q_UNUSED(parent);
+    return 5;
```
Comment (MKV21) [inline src/libclient/result/resultmodel.cpp]: Should this be constant?

File: src/mobile/qml/Results.qml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        scheduler: client.resultScheduler
+    }
+
+    delegate: ResultsDelegate {
+        headline: model.taskId + "\t" + model.secondColumn
+        onClicked: showResult(ListView.view.model.get(model.index))
+    }
+
+    function getHeaders(name) {
+        if (name === "Ping".toUpperCase()) {
```
Comment (MKV21) [inline src/mobile/qml/Results.qml]: Add other measurements. And why using "Ping".toUpperCase() and not "PING"?

**Comment by MKV21** [comment]
What you wanted to do "des mit dem rotieren, dass die result mit rotiert werden"

File: src/libclient/result/resultmodel.cpp (hunk 2 above)
Comment (monstermunchkin) [inline src/libclient/result/resultmodel.cpp]: It doesn't seem to matter as long as it's not 0.

**Pull request merged**